react/features/participants-pane/components/native/MeetingParticipantList.js

```diff
@@ -171,14 +168,12 @@ class MeetingParticipantList extends PureComponent<Props> {
  */
 function _mapStateToProps(state): Object {
     const _participantsCount = getParticipantCountWithFake(state);
-    const { remoteParticipants } = state['features/filmstrip'];
     const _showInviteButton = shouldRenderInviteButton(state);
 
     return {
         _participantsCount,
         _showInviteButton,
-        _sortedRemoteParticipants: remoteParticipants,
-        _localParticipantId: getLocalParticipant(state)?.id
+        _sortedParticipants: getSortedParticipantIds(state)
```

Comment:
Please undo this. It's returning a new array every time, thus causing re-renders, we optimized it for 500 participants, as otherwise it takes seconds to load and it really struggles during scroll.

Reply:
But this is used on the web as well (the getSortedParticipantIds), and it was optimized for 500 participants

Reply:
I don't have the time to run a 5000 test right now. Please undo it.